src/mame/drivers/aussiebyte.cpp

```diff
@@ -423,6 +423,62 @@ static SLOT_INTERFACE_START( aussiebyte_floppies )
 	SLOT_INTERFACE( "525qd", FLOPPY_525_QD )
 SLOT_INTERFACE_END
 
+
+/***********************************************************
+
+    Quickload
+
+    This loads a .COM file to address 0x100 then jumps
+    there. Sometimes .COM has been renamed to .CPM to
+    prevent windows going ballistic. These can be loaded
+    as well.
+
+************************************************************/
+
+QUICKLOAD_LOAD_MEMBER( aussiebyte_state, aussiebyte )
+{
+	uint16_t i;
+	uint8_t data;
+	uint16_t sp;
```

Comment:
Please reduce the scope of your locals and use `const` as appropriate.

Reply:
ok, I can totally get rid of the sp variable everywhere

Reply:
You can also put `data` inside the block where it's used, since it's not used outside that block anywhere and doesn't need to persist, and you can declare `i` inside each of the `for` loop control expressions, since it's not used for anything else.  Oh, and you've also got a mixture of K&R (opening brace on same line as control statement) and Allman (opening brace on line following control statement) in a single function in each place.  Please make it consistent with the rest of the file, and at least consistent within a function.

Reply:
I see, I'm re-revising it all soon